GetDKAN/dkan
# Add header cleanup functions

connects #2770

Error when adding a file to the datastore where the column header uses a mysql reserved word
```
DKAN DATASTORE: There was a problem while importing the Resource - SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'group, group1, subgroup, aian, asian, black, highlyrural, hispanic, men, missing' at line 1
```
## How to reproduce

1. Create a resource with remote file: http://janette.dkandemo.nuamsdev.com/sites/default/files/VA-OHE-NVHER_01242017_1.csv
2. Click manage datastore and 'import'
3. Confirm you see the error above

## QA Steps

Follow the steps above and confirm you do not get the error.


## Reminders
